balrogscript/task.py

```diff
@@ -7,11 +7,14 @@
 
 log = logging.getLogger(__name__)
 
+KNOWN_ACTIONS = ('submit-locale', 'submit-toplevel', 'schedule')
```

Comment:
Tiny nit: maybe move this to a constants.py file as we have in other *-workers for consistency?